vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/AbstractNumberField.java

```diff
@@ -63,6 +63,14 @@
 
     /**
      * Sets up the common logic for number fields.
+     * <p>
+     * If {@code isInitialValueOptional} is {@code true} then the initial value
+     * is used only if element has no {@code "value"} property value, otherwise
+     * element {@code "value"} property is ignored and the initial value is set.
+     * <p>
+     * If {@code isInitialValueOptional} is {@code true} then the initial value
+     * is used only if element has no {@code "value"} property value, otherwise
+     * element {@code "value"} property is ignored and the initial value is set.
```

Comment:
This paragraph is repeated two times, remove one

Reply:
Still have this one paragraph duplicated. Note that it's not about CTOR description and param description duplication. It's about the same text duplicated 2 times in the CTOR description.